README.md

```diff
@@ -117,6 +117,13 @@ Sets `(1 - sparsity)` percent of the tensor to 0, where `sparsity` is between 0
 
 > Martens, J. (2010). Deep learning via Hessian-free optimization. In *Proceedings of the 27th International Conference on Machine Learning (ICML-10)*.
 
+#### nninit.convaware(module, tensor, [{[std]}])
```

Comment:
The docs will need to be changed to `convolutionAware` (camelCase preferable for Lua).